make/all.mk

```diff
@@ -4,7 +4,6 @@ include $(INCLUDE_DIR)make.mk
 include $(INCLUDE_DIR)shell.mk
 include $(INCLUDE_DIR)help.mk
 include $(INCLUDE_DIR)repo.mk
-include $(INCLUDE_DIR)ci.mk
 include $(INCLUDE_DIR)docker.mk
 include $(INCLUDE_DIR)flux.mk
 include $(INCLUDE_DIR)tools.mk
```

Comment:
I think we should be able to replace all tooling with devbox, we can provide custom flakes for any tools that don't have binary builds available from nix packages - see for example https://github.com/mesosphere/mindthegap/blob/main/hack/flakes/flake.nix#L15-L29

Reply:
The only thing that we have now is `gh-dkp`, it is installed by `gh` itsef and always in the latest version. Wdyt about keeping it as it is?